docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -198,23 +187,23 @@ Internal plugins, roles and playbooks (artifacts used only in testing, or only t
 Documentation requirements
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-All modules and plugins MUST:
+Collections:
 
-* Include a :ref:`DOCUMENTATION <documentation-block>` block.
-* Include an :ref:`EXAMPLES <examples-block>` block (except where not relevant for the plugin type).
-* Use FQCNs when referring to modules, plugins and documentation fragments inside and outside the collection (including ``ansible.builtin`` for the listed entities from ansible-core.
-  
-When using ``version_added`` in the documentation:
+* ``MUST`` use :ref:`links and formatting macros <linking-and-other-format-macros-within-module-documentation>`.
+* ``SHOULD`` have contributor guidelines in the ``CONTRIBUTING.md`` or ``README.md`` file.
 
-* Declare the version of the collection in which the options were added -- NOT the version of Ansible.
-* If you for some reason really have to specify version numbers of Ansible or of another collection, you also have to provide ``version_added_collection: collection_name``. We strongly recommend to NOT do this.
-* Include ``version_added`` when you add new content (modules, plugins, options) to an existing collection. The values are shown in the documentation and can be useful, but you do not need to add ``version_added`` to every option, module, and plugin when creating a new collection.
+All modules and plugins:
 
-Other items:
+* ``MUST`` include a :ref:`DOCUMENTATION <documentation-block>` block.
+* ``MUST`` include an :ref:`EXAMPLES <examples-block>` block (except where not relevant for the plugin type).
+* ``MUST`` Use FQCNs when referring to modules, plugins and documentation fragments inside and outside the collection including ``ansible.builtin.`` for ansible-core.
+* ``SHOULD`` include a :ref:`RETURN <return-block>` block for modules.
```

Comment:
I would suggest to change this to MUST:
```suggestion
* ``MUST`` include a :ref:`RETURN <return-block>` block for modules and other plugins that return data.
```
While it's OK if existing collections don't update all their existing documentation IMO, this should really be a requirement for new collections.

Reply:
100% agreed